openedx/openedx-platform
# feat: Enable taxonomy/tagging feature in MFE by default

## Description

This PR sets the tagging feature as enabled by default and creates a new waffle flag, `content_tagging.disabled,` to allow the feature to be turned off.

## Testing Instructions
1. Delete the following Waffle flags using the Django admin:
  - `new_studio_mfe.use_tagging_taxonomy_list_page`
  - `content_tagging.auto`
2. Check if the tagging feature is enabled in the course authoring MFE:
  - In the course authoring home page, you should see the Taxonomy tab (http://apps.local.edly.io:2001/course-authoring/home)
  - In the Course Outline, you should see a "Manage Tags" link in the  Course Header and the Section/Subsection menus.
  - In the Unit Outline, you should see the Tags card in the sidebar
3. Create a Waffle flag with name `content_tagging.disabled` and set it to `True` to `everyone`
4. Check if the taxonomy feature is disabled in the places above
___
Private ref: [FAL-3708](https://tasks.opencraft.com/browse/FAL-3708)
